src/test/java/org/influxdb/InfluxDBTest.java

```diff
@@ -151,6 +154,66 @@ public void testWriteStringData() {
     }
 
     /**
+     * Test writing to the database using string protocol through UDP.
+     */
+    @Test
+    public void testWriteStringDataThroughUDP() {
```

Comment:
[query] 2016/11/09 01:24:10 CREATE DATABASE udp
[httpd] 10.140.202.187 - admin [09/Nov/2016:01:24:10 +0000] "POST /query?p=%5BREDACTED%5D&q=CREATE+DATABASE+%22udp%22&u=admin HTTP/1.1" 200 46 "-" "okhttp/3.4.2" 376d3267-a61b-11e6-8703-000000000000 23125
[query] 2016/11/09 01:24:10 SELECT * FROM udp.autogen.cpu GROUP BY *
[httpd] 10.140.202.187 - admin [09/Nov/2016:01:24:10 +0000] "GET /query?db=udp&p=%5BREDACTED%5D&q=SELECT+%2A+FROM+cpu+GROUP+BY+%2A&u=admin HTTP/1.1" 200 46 "-" "okhttp/3.4.2" 37784cb0-a61b-11e6-8705-000000000000 1319
[query] 2016/11/09 01:24:10 DROP DATABASE udp
[httpd] 10.140.202.187 - admin [09/Nov/2016:01:24:10 +0000] "POST /query?p=%5BREDACTED%5D&q=DROP+DATABASE+%22udp%22&u=admin HTTP/1.1" 200 46 "-" "okhttp/3.4.2" 377e834d-a61b-11e6-8706-000000000000 40067
[udp] 2016/11/09 01:24:11 failed to write point batch to database "udp": database not found: udp